environments/auth/auth-with-federated/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "private": true,
-  "name": "auth-with-email-environment",
+  "name": "auth-with-federated-environment",
```

Comment:
This typo actually broke Angular e2e, because it would think `environments/auth/auth-with-federated` and `environments/auth/auth-with-email` are the same directory because they have the same `name` in their `package.json`. 

This was interesting because we were importing environments through explicit relative path and not through `node_modules` 🤯